Containerfile

```diff
@@ -1,19 +1,18 @@
-FROM registry.access.redhat.com/ubi9/nodejs-18 as yarn_builder
+FROM registry.access.redhat.com/ubi9/nodejs-18 as npm_builder
 # Become root before installing anything
 USER root
 RUN dnf update -y && dnf clean all
-RUN npm install --global yarn
 
```

Comment:
I'm so glad we are moving away from yarn. Installing it seems simple here, but building it downstream is a PITA.